src/levelSelect.cpp

```diff
@@ -153,6 +155,25 @@ void LevelSelect::onFixedUpdate()
             gui->hide();
             disable();
         }
+
+        if (controls[n].unknown5.getDown() || controls[n].unknown4.getDown())
```

Comment:
Rename the keys from unknown to replay_fuel and replay_time.

Reply:
Yes that would be better.
Idea for the future: It might be better to have an array of KeyState's in PlayerControlsState instead of a list of instances and then have alliases to single KeyStates in the array.